devtools-ci-index.yaml

```diff
@@ -871,4 +871,4 @@
             git_repo: openshiftio
             ci_project: 'devtools'
             ci_cmd: '/bin/bash cico_build_deploy.sh'
-            svc_name: catapult
+            svc_name: launchpad-missioncontrol
```

Comment:
As long as, you define a service with this name in Openshift. This looks good to me.